tools/scripts-config-react-webpack/config/webpack.config.js

```diff
@@ -379,6 +381,18 @@ module.exports = ({ getUserConfig, mode }) => {
 						filename: getFileNameForExtension('css', cssPrefix),
 						chunkFilename: getFileNameForExtension('css', cssPrefix),
 					}),
+				isEnvProduction &&
+					!!sentryConfig &&
+					new SentryWebpackPlugin({
+						// see https://docs.sentry.io/product/cli/configuration/ for details
+						org: 'talend-0u',
```

Comment:
org should be a parameter with this value as default value.

Reply:
changed to `sentryConfig.org || process.env.SENTRY_ORG || 'talend-0u'`